soh/src/overlays/misc/ovl_kaleido_scope/z_lmap_mark.c

```diff
@@ -56,122 +64,98 @@ void PauseMapMark_DrawForDungeon(PlayState* play) {
     PauseMapMarkInfo* markInfo;
     f32 scale;
     s32 i = 0;
-
```

Comment:
This function is largely reworked for the same reasons as the minimap map mark function.